src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -270,10 +271,19 @@ class editIndex extends viewModelBase {
     }
 
     private fetchCustomAnalyzers() {
+        let analyzersList = new Array<string>();
+        
         new getCustomAnalyzersCommand(this.activeDatabase(), true)
```

Comment:
let's wrap this with $.when to have consistent order, right now order is not deterministic, right?

Reply:
done